src/components/ListOrgs.vue

```diff
@@ -250,6 +336,23 @@ const tableData = computed(() => {
   });
 });
 
+const ShowCode = (selectedOrg) => {
+  const collections = [allDistricts.value, allSchools.value, allClasses.value, allGroups.value];
+
+  for (const collection of collections) {
+    const match = collection?.find((item) => item.id === selectedOrg && item.currentActivationCode);
+    if (match) {
+      activationCode.value = match.currentActivationCode;
+      isDialogVisible.value = true;
+      break;
+    }
+  }
```

Comment:
I think we can retrieve the activation code without querying all of the orgs:

```suggestion
  const orgInfo = await fetchDocById(activeOrgType.value, orgId);
  if (orgInfo?.currentActivationCode) {
    activationCode.value = orgInfo.currentActivationCode;
    isDialogVisible.value = true;
  }
```

Reply:
Let me try